### PR TITLE
[14.0][MIG] l10n_ch_pain_base

### PR DESCRIPTION
**Pull request by michelerusti**
Since #591 is inactive I've rebased as requested with the latest 14.0 version

### CODE REVIEW

**Comment by michelerusti** [comment]
Looks like a different test fails this time! 
The check in error
`self.assertEqual(invoice.amount_by_group[0][1], 0.35000000000000003)`

Since the variable in question is defined as
`amount = fields.Float(required=True, digits=(16, 4), default=0.0)`

I'm not sure it is needed at all since the digits are limited to 4 decimals, am i missing something?

**Comment by michelerusti** [comment]
I've fixed the failing test, let me know if it's all good!

**Comment by michelerusti** [comment]
@i-vyshnevska This PR has been stuck for a while, is it possible to carry it on?

**Comment by michelerusti** [comment]
@i-vyshnevska I'm sorry to ping this PR again, if some edits are needed let me know!

File: ebill_paynet/tests/test_invoice_message_b2b.py [hunk 1]
```diff
@@ -108,8 +108,8 @@ def test_invoice_qr_rounding(self, get_ic_ref):
         invoice.action_post()
         invoice.payment_reference = "1234567890"
         invoice.partner_bank_id = self.partner_bank.id
-        # amount inside is not rounded
-        self.assertEqual(invoice.amount_by_group[0][1], 0.35000000000000003)
+        # amount inside is not rounded but the field goes 4 decimals deep
+        self.assertAlmostEqual(invoice.amount_by_group[0][1], 0.35000000000000003, 4)
```
Comment (i-vyshnevska) [inline ebill_paynet/tests/test_invoice_message_b2b.py:112]: interesting why this test starts to fail current master branch seems ok, addon doesn't introduce related changes

Comment (davidwul) [reply ebill_paynet/tests/test_invoice_message_b2b.py:112]: @i-vyshnevska 
Sorry for a beginer question, is there anthing to do yet on the PR?

Comment (michelerusti) [reply ebill_paynet/tests/test_invoice_message_b2b.py:112]: > @i-vyshnevska Sorry for a beginer question, is there anthing to do yet on the PR?

I'm wondering the same thing

Comment (michelerusti) [reply ebill_paynet/tests/test_invoice_message_b2b.py:112]: @i-vyshnevska do you thing is it better to close this PR or does it still needs changes?

File: l10n_ch_pain_base/views/bank_payment_line.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+  © 2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+<odoo>
+
+    <record id="bank_payment_line_form" model="ir.ui.view">
+        <field name="name">l10n_ch_sepa.bank.payment.line.form</field>
+        <field name="model">bank.payment.line</field>
```
Comment (Darknroses) [inline l10n_ch_pain_base/views/bank_payment_line.xml:10]: This model has been removed from account_payment_order

https://github.com/OCA/bank-payment/commit/b8d818967bc221f1663bf8be99ffa4165f321414#diff-bc6e2817a362706bd1825d6d91ccc21ce12c159dd8ed779ff169c92bf3ad7512

**Review by Darknroses — CHANGES_REQUESTED** [review]

**Pull request closed without merging**